udata/templates/dataset/display.html

```diff
@@ -33,15 +33,9 @@
     </a>
 </div>
 {% endif %}
+
 {% if sysadmin %}
-<div class="btn-group btn-group-xs">
-    <button type="button"
-        class="btn btn-success featured {% if dataset.featured %}active{% endif %}"
-        data-api-url="{{ url_for('api.dataset_featured', dataset=dataset) }}">
-        <span class="glyphicon glyphicon-bullhorn"></span>
-        {{ _('Featured') }}
-    </button>
-</div>
+<featured dataset-id="{{ dataset.id }}" :featured="{% if dataset.featured %}true{% else %}false{% endif %}"></featured>
```

Comment:
Or even simpler with `:featured="{{ dataset.featured|to_json }}"` (it should work) ;)